### PR TITLE
[TVMC] Python Scripting Init Files

### PR DESCRIPTION
**Pull request by CircleSpin**
This PR adds to two init files to make tvmc cleaner for running python scripts. For example, instead of running tvmc.runner.run_module() you can now run tvmc.run()

### CODE REVIEW

**Comment by CircleSpin** [comment]
@jwfromm @leandron @mdw-octoml

File: python/tvm/driver/tvmc/__init__.py [hunk 1]
```diff
@@ -21,3 +21,6 @@
 from . import autotuner
 from . import compiler
 from . import runner
+from .frontends import load_model as load
+from .compiler import compile_model as compile
+from .runner import run_module as run
```
Comment (mdw-octoml) [inline python/tvm/driver/tvmc/__init__.py:26]: Note need for newline at end of file

**Review by mdw-octoml — APPROVED** [review]
LGTM!

**Review by leandron — COMMENTED** [review]
Is the use case here that TVMC is used as an API within python scripts? it look OK to me, since it is part of TVM.

Can we think of a few test cases to add, to cover this new entry-point for tvmc as API?

cc @comaniac

**Comment by jwfromm** [comment]
@leandron yeah we think TVMC has an excellent set of utilities to make it a simplified python API for new users, kind of like keras. You did an awesome job packing tons of functionality into a few key function calls, but some of them need some small changes to better accommodate access from Python. @CircleSpin is working on making these changes.

**Comment by CircleSpin** [comment]
I've modified the test files so that they use new the importing syntax. Does this look good to you @leandron ?

**Review by comaniac — APPROVED** [review]
LGTM

**Review by leandron — APPROVED** [review]
Thanks @CircleSpin

**Comment by jwfromm** [comment]
@merrymercy this PR is failing with this error message `AttributeError: module 'tvm.auto_scheduler._ffi_api' has no attribute 'RandomModel'`, which almost certainly isn't caused by the changes made. Do you know what's going on?

**Comment by comaniac** [comment]
Just a guess. Is that due to circular dependency?

**Comment by jwfromm** [comment]
Thanks @CircleSpin @leandron and @comaniac. This is now merged.

**Pull request merged**